Handlers/ReceiptsHandler.fs

```diff
@@ -0,0 +1,21 @@
+namespace ExpenseReports.Handlers
+
+open System.IO
+open Microsoft.AspNetCore.Http
+open Giraffe.HttpHandlers
+open Giraffe.Tasks
+
+module ReceiptHandler =
+    let receiptUploadHandler =
```

Comment:
This will gives us ReceiptsHandler.receiptUploadHandler, maybe we could shorten it to Receipts.uploadHandler?